backend/Squirrel.Core/Squirrel.Core.WebAPI/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -25,6 +28,19 @@ public static void RegisterCustomServices(this IServiceCollection services)
         services.AddScoped<JwtIssuerOptions>();
         services.AddScoped<IJwtFactory, JwtFactory>();
         services.AddScoped<IAuthService, AuthService>();
+
+        services.AddMongoDbService(configuration);
```

Comment:
just use `AddMongoDbService` in `Program` and then we don't need `services.AddMongoDbService(configuration);` and `IConfiguration configuration` in    `RegisterCustomServices`